src/services/CalculatorProcess.js

```diff
@@ -35,7 +37,7 @@ function CalculatorProcess(content, visor) {
 		memory = memory.slice(0, -sLength);
 
 		for (let i = 0; i < sLength; i++) {
```

Comment:
why do you need this for?

Reply:
yea you're right don't make sense now, this was before i create `conversion()` and i didn't realized that i could remove it

Reply:
ah OK ;)